pkg/endpoint/endpoint.go

```diff
@@ -109,6 +109,8 @@ var _ notifications.RegenNotificationInfo = &Endpoint{}
 type Endpoint struct {
 	owner regeneration.Owner
 
+	proxy EndpointProxy
```

Comment:
Can you put this after the `ID`? I know this sounds like a weird request but sometimes it is possible to derive the endpoint ID from stack traces, works because ID is one of the first fields of this structure.